Lesson31/Gokhan/demo-app/src/shared-components/typography/BodyM/styles.css

```diff
@@ -0,0 +1,9 @@
+.bodyM {
+  width: 55px;
```

Comment:
Never set height and with for text elements. In most cases, you cannot know beforehand how large the text content will be.